packages/client/e2e/mpe/test-a.spec.ts

```diff
@@ -78,17 +82,12 @@ async function exportMpeRoomToMtvRoom({
  * -UserA shouldn't be able to see the deleted track on both his devices
  */
 test('Create MPE room', async ({ browser }) => {
-    const { page } = await setupAndGetUserPage({
+    const { page, context } = await setupAndGetUserPage({
         browser,
         knownSearches,
-        userIndex: 0,
     });
 
-    const { page: pageB } = await setupAndGetUserPage({
-        browser,
-        knownSearches,
-        userIndex: 0,
-    });
+    const { page: pageB } = await createNewTabFromExistingContext(context);
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏Nice! It's way better than choosing between four predefined users.